e2e/_suites/fleet/fleet.go

```diff
@@ -1127,6 +1127,81 @@ func (fts *FleetTestSuite) upgradeAgent(version string) error {
 	return nil
 }
 
+func (tfs *FleetTestSuite) checkDataStream() error {
```

Comment:
Consistency in the receiver name

```suggestion
func (fts *FleetTestSuite) checkDataStream() error {
```